.gitignore

```diff
@@ -3,7 +3,7 @@
 *.swp
 *.DS_Store
 /werlog.js
-/data
+/runs
```

Comment:
It is not the primary place for training data anymore. Most of the files in there are repository owned now.